vraptor-core/src/main/java/br/com/caelum/vraptor/core/JstlLocalization.java

```diff
@@ -75,7 +73,7 @@ private ResourceBundle extractUnsafeBundle(Object bundle, Locale locale) {
 			String baseName = firstNonNull((String) bundle, DEFAULT_BUNDLE_NAME);
 
 			try {
-				return ResourceBundle.getBundle(baseName, locale);
+				return ResourceBundle.getBundle(baseName, locale, new UTF8Control());
```

Comment:
Turini,
Sorry for the late reply.
I agree with @csokol @cpfeiffer @lucascs that changing the ISO-8859-1 default to utf8 is not good solution.
I need to learn about CDI to be able give expert opinion.